File: Arrays & Strings/002 Strings/First Letter to Appear Twice/Solution.java
class Solution {
    public char repeatedCharacter(String s) {
        int count[]= new int[26];
        
        for(int i=0;i<s.length();i++) {
            if(count[s.charAt(i)-'a']==1)
                return s.charAt(i);
            else
                count[s.charAt(i)-'a']++; 
        }

        return 'a';
    }
}

File: Arrays & Strings/002 Strings/First Letter to Appear Twice/Solution.kt
class Solution {
    fun repeatedCharacter(s: String): Char {
        val count= IntArray(26)
        
        for(c in s)
            if(count[c-'a']==1)
                return c
            else
                count[c-'a']++

        return 'a'
    }
}
